src/test/pool_tests.cpp

```diff
@@ -54,33 +54,40 @@ BOOST_AUTO_TEST_CASE(basic_allocating)
     PoolResourceTester::CheckAllDataAccountedFor(resource);
     BOOST_TEST(1 == PoolResourceTester::FreeListSizes(resource)[1]);
     BOOST_TEST(expected_bytes_available == PoolResourceTester::AvailableMemoryFromChunk(resource));
+    // the system allocates 32 bytes for an 8-byte allocation
+    BOOST_TEST(32 == resource.SystemBytes());
```

Comment:
This test may be somewhat fragile; it depends on `MallocUsage(8) == 32`. Maybe the test condition here should be `8 <= resource.SystemBytes()`, but I think that doesn't test enough.

Reply:
CI failed here (on 32-bit), so I force-pushed to make the test more precise.